src/Agent/NewRelic/Agent/Core/Utilities/FileWrapper.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.IO;
+
+namespace NewRelic.Agent.Core.Utilities
+{
+    /// <summary>
+    /// Wraps some File methods to allow for unit testing
+    /// </summary>
+    public interface IFileWrapper
```

Comment:
I was going to suggest that we have a single interface for mocking file system calls, but it doesn't seem like we need that in too many places, so this is probably fine. :)

Reply:
Yeah, I thought maybe we already had one somewhere but didn't see anything. This is easy enough to extend if/when we need it elsewhere.